lib/dev-server.js

```diff
@@ -24,6 +24,7 @@ async function devServer(options) {
     app.use(nollupDevMiddleware(app, config, {
         hot: options.hot,
         verbose: options.verbose,
+        headers: options.headers,
```

Comment:
I'd remove this since it doesn't do anything, and Object.assign can take undefined as an argument.